source/authentication/email-password.txt

```diff
@@ -511,6 +511,12 @@ email/password authentication, see the documentation for the Realm SDKs:
       To register or log in an email/password user from the .NET Client SDK, see the
       :ref:`.NET SDK guide to email/password authentication <dotnet-login-email-password>`.
 
+   .. tab::
+      :tabid: flutter
+
+      To register or log in an email/password user from the Flutter Client SDK, see the
+      :ref:`Flutter SDK guide to email/password authentication <flutter-login-email-password>`.
+
```

Comment:
This is a bit selfish, but... would you mind adding a kotlin tab in these places as well? Just seems easier for you to add one, and since it's GA Monday, it probably deserves a mention even more than flutter!

Reply:
got u b